File: gzoom2-fe/app/src/app/api/model/score.ts
import { AcctgTransEntry } from "./acctgTransEntry";
import { CustomTimePeriod } from "./customTimePeriod";
import { GlAccount } from "./glAccount";
import { GlFiscalType } from "./glFiscalType";

/**
 * Model of Score
 */
export class Score {
    public glAccount?: GlAccount;
    public glFiscalType?: GlFiscalType;
    public customTimePeriod?: CustomTimePeriod;
    public acctgTransEntry?: AcctgTransEntry;
}

File: gzoom2-be/persistence-mybatis/src/main/java/it/mapsgroup/gzoom/mybatis/dto/Score.java
package it.mapsgroup.gzoom.mybatis.dto;

public class Score {
    private CustomTimePeriod customTimePeriod;
    private GlAccount glAccount;
    private AcctgTransEntry acctgTransEntry;
    private GlFiscalType glFiscalType;

    public CustomTimePeriod getCustomTimePeriod() {
        return customTimePeriod;
    }

    public void setCustomTimePeriod(CustomTimePeriod customTimePeriod) {
        this.customTimePeriod = customTimePeriod;
    }

    public AcctgTransEntry getAcctgTransEntry() {
        return acctgTransEntry;
    }

    public void setAcctgTransEntry(AcctgTransEntry acctgTransEntry) {
        this.acctgTransEntry = acctgTransEntry;
    }

    public GlFiscalType getGlFiscalType() {
        return glFiscalType;
    }

    public void setGlFiscalType(GlFiscalType glFiscalType) {
        this.glFiscalType = glFiscalType;
    }


    public GlAccount getGlAccount() {
        return glAccount;
    }

    public void setGlAccount(GlAccount glAccount) {
        this.glAccount = glAccount;
    }
}
